### DISCUSSION TITLE
How to start and where to put GeneralSetting?

### DISCUSSION THREAD

**Discussion by shiroamada** [spatie/laravel-settings#14] [Q&A] [answered]
Thanks for the great package. When I read through the documentation, 
I am stuck at the "GeneralSettings", where should I put this file as good practise? 
I put at `app/Http/Settings/GeneralSettings.php` everything is working fine, just want to check am I doing correctly?

Normally Documentation will give suggestion where to create the file, but I can't see from the step, sorry for the beginner question but I just want to make sure I start it correctly.

Thanks!

**Comment by rubenvanassche** [comment, marked as answer]
In bigger application we have multiple contexts for each domain, so for example a `Blog`, `User`. and `Invoice` context. In those cases we put settings files in each context so for the `Blog` context that would be: `app/Context/Blog/Settings/BlogSettings.php`. In Smaller applications I would put them within the `app` directory so in your case `app/Settings/GeneralSettings.php`.

But it is possible to put these files wherever you want, the above is more of a personal choice from the Spatie team.
